examples/multi-tenancy-with-teams/outputs.tf

```diff
@@ -2,3 +2,8 @@ output "ssp_output" {
   description = "SSP module outputs"
   value       = module.aws-eks-accelerator-for-terraform
 }
+
+output "configure_kubectl" {
+  description = "Configure kubectl: make sure you're logged in with the correct AWS profile and run the following command to update your kubeconfig"
+  value       = "aws eks --region ${data.aws_region.current.id} update-kubeconfig --name ${module.aws-eks-accelerator-for-terraform.eks_cluster_id}"
+}
```

Comment:
Add outputs for individual teams being created here as the platform team and application team members each need to pass a `--role-arn` flag to the `update-kubeconfig` command with the respective role we create for them.